packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -189,7 +189,7 @@ async function verifyRequestState(options: AuthenticateRequestOptions, token: st
 export const isSatelliteAndNeedsSyncing: InterstitialRule = options => {
   const { clientUat, isSatellite, searchParams, secretKey, apiKey, userAgent } = options;
 
-  const key = secretKey || apiKey;
+  const key = (secretKey || apiKey) as string;
```

Comment:
isn't wrong to assume this will work ? How about this instead ? 
```suggestion
  const key = secretKey || apiKey || '';
```

Reply:
Changed all the occurrences!